src/sbml/SpeciesReference.cpp

```diff
@@ -982,9 +982,7 @@ SpeciesReference::removeChildObject(const std::string& elementName, const std::s
 
   if (elementName == "stoichiometryMath")
   {
-    StoichiometryMath* t = getStoichiometryMath();
-    if (unsetStoichiometryMath() == LIBSBML_OPERATION_SUCCESS)
-      return t;
+    unsetStoichiometryMath();
```

Comment:
return ?

Reply:
the return is at the end of the function ... so i didn't see a need to duplicate it here, can change it though